.travis.yml

```diff
@@ -0,0 +1,57 @@
+sudo: false
```

Comment:
Looks like I missed this before, I was going to add it to .NET before I read about it not being needed.
From <https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration>, `If you currently specify sudo: false in your .travis.yml, we recommend removing that configuration soon.`
```suggestion
```